### PR TITLE
Add separate dependencies bib

### PR DESCRIPTION
**Pull request by nilsdeppe**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: Metadata.yaml [hunk 1]
```diff
@@ -365,4 +365,4 @@ References:
     them in publications.
   BibliographyFile: docs/References.bib
   List:
-    - charmpp
+    - Charmpp1
```
Comment (nilsvu) [inline Metadata.yaml:370]: Add the other refs?

**Comment by nilsdeppe** [comment]
@nilsvu this is breaking in one of the python scripts because it only handles one bib file, I think. I'm not sure how we want to handle that...

File: Metadata.yaml [hunk 2]
```diff
@@ -365,4 +365,4 @@ References:
     them in publications.
   BibliographyFile: docs/References.bib
```
Comment (nilsvu) [inline Metadata.yaml]: Just change this to the new file you added to fix the CI error

**Review by nilsvu — CHANGES_REQUESTED** [review]
Haven't checked the new references, will do that asap

**Comment by nilsvu** [comment]
Here's what will be sent to Zenodo based on the input:

- Klaus Iglberger, Georg Hager, Jan Treibig, and Ulrich Rüde. High performance smart
  expression template math libraries. In 2012 International Conference on High Performance
  Computing & Simulation (HPCS), volume, 367-373. 2012. doi:10.1109/HPCSim.2012.6266939.
- 'Klaus Iglberger, Georg Hager, Jan Treibig, and Ulrich Rüde. Expression templates
  revisited: a performance analysis of current methodologies. SIAM Journal on Scientific
  Computing, 34(2):C42-C69, 2012. URL: https://doi.org/10.1137/110830125, arXiv:https://doi.org/10.1137/110830125,
  doi:10.1137/110830125.'
- 'Laxmikant Kale, Bilge Acun, Seonmyeong Bak, Aaron Becker, Milind Bhandarkar, Nitin
  Bhat, Abhinav Bhatele, Eric Bohm, Cyril Bordage, Robert Brunner, Ronak Buch, Sayantan
  Chakravorty, Kavitha Chandrasekar, Jaemin Choi, Michael Denardo, Jayant DeSouza,
  Matthias Diener, Harshit Dokania, Isaac Dooley, Wayne Fenton, Juan Galvez, Fillipo
  Gioachin, Abhishek Gupta, Gagan Gupta, Manish Gupta, Attila Gursoy, Vipul Harsh,
  Fang Hu, Chao Huang, Narain Jagathesan, Nikhil Jain, Pritish Jetley, Prateek Jindal,
  Raghavendra Kanakagiri, Greg Koenig, Sanjeev Krishnan, Sameer Kumar, David Kunzman,
  Michael Lang, Akhil Langer, Orion Lawlor, Chee Wai Lee, Jonathan Lifflander, Karthik
  Mahesh, Celso Mendes, Harshitha Menon, Chao Mei, Esteban Meneses, Eric Mikida, Phil
  Miller, Ryan Mokos, Venkatasubrahmanian Narayanan, Xiang Ni, Kevin Nomura, Sameer
  Paranjpye, Parthasarathy Ramachandran, Balkrishna Ramkumar, Evan Ramos, Michael
  Robson, Neelam Saboo, Vikram Saletore, Osman Sarood, Karthik Senthil, Nimish Shah,
  Wennie Shu, Amitabh B. Sinha, Yanhua Sun, Zehra Sura, Ehsan Totoni, Krishnan Varadarajan,
  Ramprasad Venkataraman, Jackie Wang, Lukasz Wesolowski, Sam White, Terry Wilmarth,
  Jeff Wright, Joshua Yelon, and Gengbin Zheng. The Charm++ Parallel Programming System.
  Aug 2019. URL: https://charm.cs.illinois.edu, doi:10.5281/zenodo.3370873.'
- 'Laxmikant Kale, Bilge Acun, Seonmyeong Bak, Aaron Becker, Milind Bhandarkar, Nitin
  Bhat, Abhinav Bhatele, Eric Bohm, Cyril Bordage, Robert Brunner, Ronak Buch, Sayantan
  Chakravorty, Kavitha Chandrasekar, Jaemin Choi, Michael Denardo, Jayant DeSouza,
  Matthias Diener, Harshit Dokania, Isaac Dooley, Wayne Fenton, Zane Fink, Juan Galvez,
  Pathikrit Ghosh, Fillipo Gioachin, Abhishek Gupta, Gagan Gupta, Manish Gupta, Attila
  Gursoy, Vipul Harsh, Fang Hu, Chao Huang, Narain Jagathesan, Nikhil Jain, Pritish
  Jetley, Prateek Jindal, Raghavendra Kanakagiri, Greg Koenig, Sanjeev Krishnan, Sameer
  Kumar, David Kunzman, Michael Lang, Akhil Langer, Orion Lawlor, Chee Wai Lee, Jonathan
  Lifflander, Karthik Mahesh, Celso Mendes, Harshitha Menon, Chao Mei, Esteban Meneses,
  Eric Mikida, Phil Miller, Ryan Mokos, Venkatasubrahmanian Narayanan, Xiang Ni, Kevin
  Nomura, Sameer Paranjpye, Parthasarathy Ramachandran, Balkrishna Ramkumar, Evan
  Ramos, Michael Robson, Neelam Saboo, Vikram Saletore, Osman Sarood, Karthik Senthil,
  Nimish Shah, Wennie Shu, Amitabh B. Sinha, Yanhua Sun, Zehra Sura, Justin Szaday,
  Ehsan Totoni, Krishnan Varadarajan, Ramprasad Venkataraman, Jackie Wang, Lukasz
  Wesolowski, Sam White, Terry Wilmarth, Jeff Wright, Joshua Yelon, and Gengbin Zheng.
  UIUC-PPL/charm: Charm++ version 7.0.0. October 2021. URL: https://doi.org/10.5281/zenodo.5597907,
  doi:10.5281/zenodo.5597907.'
- 'Laxmikant V Kale and Sanjeev Krishnan. Charm++: parallel programming with message-driven
  objects. In Gregory V. Wilson and Paul Lu, editors, Parallel programming using C++,
  pages 175–213. The MIT Press, 1996.'
- M. Galassi and others. GNU Scientific Library Reference Manual. Network Theory Ltd.,
  3 edition, 2009. ISBN 9780954612078.
- The HDF Group. Hierarchical Data Format, version 5. 1997-2023. https://www.hdfgroup.org/HDF5/.
- M. Reinecke and D. S. Seljebotn. Libsharp - spherical harmonic transforms revisited.
  \aap , 554:A112, June 2013. arXiv:1303.4945, doi:10.1051/0004-6361/201321494.
- 'Alexander Heinecke, Greg Henry, Maxwell Hutchinson, and Hans Pabst. LIBXSMM: Accelerating
  small matrix multiplications by runtime code generation. In Proceedings of the International
  Conference for High Performance Computing, Networking, Storage and Analysis, SC
  ''16, 1-11. IEEE Press, 2016.'
- 'J. D. Hunter. Matplotlib: a 2d graphics environment. Computing in Science & Engineering,
  9(3):90–95, 2007. doi:10.1109/MCSE.2007.55.'
- 'Thomas A Caswell, Michael Droettboom, Antony Lee, John Hunter, Eric Firing, Elliott
  Sales de Andrade, Tim Hoffmann, David Stansby, Jody Klymak, Nelle Varoquaux, Jens
  Hedegaard Nielsen, Benjamin Root, Ryan May, Phil Elson, Darren Dale, Jae-Joon Lee,
  Jouni K. Seppänen, Damon McDougall, Andrew Straw, Paul Hobson, Christoph Gohlke,
  Tony S Yu, Eric Ma, Adrien F. Vincent, Steven Silvester, Charlie Moad, Nikita Kniazev,
  hannah, Elan Ernest, and Paul Ivanov. Matplotlib/matplotlib: REL: v3.3.0. July 2020.
  URL: https://doi.org/10.5281/zenodo.3948793, doi:10.5281/zenodo.3948793.'
- 'Charles R. Harris, K. Jarrod Millman, Stéfan J. van der Walt, Ralf Gommers, Pauli
  Virtanen, David Cournapeau, Eric Wieser, Julian Taylor, Sebastian Berg, Nathaniel
  J. Smith, Robert Kern, Matti Picus, Stephan Hoyer, Marten H. van Kerkwijk, Matthew
  Brett, Allan Haldane, Jaime Fernández del Río, Mark Wiebe, Pearu Peterson, Pierre
  Gérard-Marchant, Kevin Sheppard, Tyler Reddy, Warren Weckesser, Hameer Abbasi, Christoph
  Gohlke, and Travis E. Oliphant. Array programming with NumPy. Nature, 585(7825):357–362,
  September 2020. URL: https://doi.org/10.1038/s41586-020-2649-2, doi:10.1038/s41586-020-2649-2.'
- 'Utkarsh Ayachit. The ParaView Guide: A Parallel Visualization Application. Kitware,
  Inc., Clifton Park, NY, USA, 2015. ISBN 1930934300.'
- 'J. Ahrens, Berk Geveci, and C. Law. ParaView: An end-user tool for large-data visualization.
  Elsevier, 2005. ISBN 9780123875822.'
- Wenzel Jakob, Jason Rhinelander, and Dean Moldovan. Pybind11 – seamless operability
  between c++11 and python. 2017. https://github.com/pybind/pybind11.
- 'Jesse Beder, Matthew Woehlke, Jens Breitbart, Scott Wolchok, Azamat H. Hackimov,
  Jamie Snape, Oliver Hamlet, Paul Novotny, Raul Tambre, Stefan Reinhold, Alain Vaucher,
  Alexander Zaitsev, Alexander Anokhin, Alexander Karatarakis, Andy Maloney, Antony
  Polukhin, Craig M. Brandenburg, Dan Ibanez, Denis Gladkikh, Florian Eich, Guillaume
  Dumont, Haydn Trigg, Jim King, Joel Frederico, Jonathan Hamilton, Joseph Langley,
  Lassi Hämäläinen, Matt Blair, Michael Welsh Duggan, Olli Wang, Patrick Stotko, Peter
  Levine, Petr Bena, Rodrigo Hernandez Cordoba, Ryan Schmidt, Simon Gene Gottlieb,
  Franz Prilmeier, Tanki Zhang, Tatsuyuki Ishi, Ted Lyngmo, Victor Mataré, Michael
  Konečný, and USDOE. Yaml-cpp. 9 2009. URL: https://www.osti.gov/biblio/1881863,
  doi:10.11578/dc.20220817.13.'

**Comment by nilsdeppe** [comment]
Looks good to me :shrug:

**Review by nilsvu — APPROVED** [review]

**Pull request merged**